tests/test_xvfb.py

```diff
@@ -201,3 +207,28 @@ def test_fixt(fixt):
     """)
     result = testdir.runpytest()
     assert result.ret == 0
+
+
+@pytest.mark.skipif(not xauth_available, reason='no xauth')
+def test_xvfb_with_xauth(testdir):
+    original_auth = os.getenv('XAUTHORITY')
+    testdir.makeini("""
+        [pytest]
+        xvfb_xauth = True
+    """)
+    testdir.makepyfile("""
+        import os
+
+        def test_xauth():
+            print('\\nXAUTHORITY: ' + os.environ['XAUTHORITY'])
+            assert os.access(os.environ['XAUTHORITY'], os.R_OK)
+    """)
+    result = testdir.runpytest('-s')
+    # Get and parse the XAUTHORITY: line
+    authline = next(l for l in result.outlines if l.startswith('XAUTHORITY:'))
+    authfile = authline.split(' ', 1)[1]
+
+    assert result.ret == 0
+    # Make sure the authfile is deleted
+    assert not os.path.isfile(authfile)
```

Comment:
Shouldn't make a difference, I kind of took inspiration from [the PR I wrote](https://github.com/ponty/PyVirtualDisplay/pull/17/files#diff-9a88591d717ee5a313cc116ffc581011R31), and there I just went with the first thing that came to my mind (checking `os.path.isfile` in the test, checking `not os.path.isfile` after the test). Here I went with `os.access` in the test (could also be `os.path.isfile`), and kept the `not os.path.isfile`. \</bikeshedding\> :stuck_out_tongue_winking_eye: 

...no idea, will change it.